### DISCUSSION TITLE
:java.lang.OutOfMemoryError: Java heap space on 157 page files

### DISCUSSION THREAD

**Discussion by Alex-0529** [red6/pdfcompare#144] [Q&A]
I getting Exceptions when comparing 157 page files

i using 1.2.0 version

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

1.1.62 worked fine

when use PDF Studio for same files on both versions i am getting 

[PDF Compare - Java heap aout of memory.txt](https://github.com/red6/pdfcompare/files/14500168/PDF.Compare.-.Java.heap.aout.of.memory.txt)

my heap size:
java -XX:+PrintFlagsFinal -version | findstr /i "HeapSize PermSize ThreadStackSize"
     intx CompilerThreadStackSize                   = 0                                   {pd product}
    uintx ErgoHeapSizeLimit                         = 0                                   {product}
    uintx HeapSizePerGCThread                       = 87241520                            {product}
    uintx InitialHeapSize                          := 1067450368                          {product}
    uintx LargePageHeapSizeThreshold                = 134217728                           {product}
    uintx MaxHeapSize                              := 4188012544                          {product}
     intx ThreadStackSize                           = 0                                   {pd product}
     intx VMThreadStackSize                         = 0                                   {pd product}
openjdk version "1.8.0_242"
OpenJDK Runtime Environment (Zulu 8.44.0.10-SA-win64) (build 1.8.0_242-b20)
OpenJDK 64-Bit Server VM (Zulu 8.44.0.10-SA-win64) (build 25.242-b20, mixed mode)

**Comment by finsterwalder** [comment]
Can you please retry with version 1.2.2?

**Reply by finsterwalder** [reply]
Closed, because there was no further reply...
